widget/entry_test.go

```diff
@@ -2142,11 +2112,6 @@ func setupSelection(t *testing.T, reverse bool) (*widget.Entry, fyne.Window) {
 	return e, window
 }
 
-func teardownImageTest(w fyne.Window) {
-	w.Close()
-	test.NewApp()
```

Comment:
This line is no longer called from many tests. Is that ok?

Reply:
Yes, that is expected. I wired up `setupImageTest` and `setupPasswordTest` to both use `NewTempWindow` and create a new application when cleaning up after the test.